### PR TITLE
[zh-cn]SYNC install-kubectl

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh-cn/docs/tasks/tools/install-kubectl-linux.md [hunk 1]
```diff
@@ -64,15 +64,15 @@ The following methods exist for installing kubectl on Linux:
...
-   -->
-   如需下载某个指定的版本，请用指定版本号替换该命令的这一部分：
-   `$(curl -L -s https://dl.k8s.io/release/stable.txt)`。
+For example, to download version {{< param "fullversion" >}} on Linux, type:
+-->
+如需下载某个指定的版本，请用指定版本号替换该命令的这一部分：
+`$(curl -L -s https://dl.k8s.io/release/stable.txt)`。
 
-   例如，要在 Linux 中下载 {{< param "fullversion" >}} 版本，请输入：
+例如，要在 Linux 中下载 {{< param "fullversion" >}} 版本，请输入：
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/tools/install-kubectl-linux.md:75]:
```suggestion
   例如，要在 Linux 中下载 {{< param "fullversion" >}} 版本，请输入：
```

File: content/zh-cn/docs/tasks/tools/install-kubectl-linux.md [hunk 2]
```diff
@@ -64,15 +64,15 @@ The following methods exist for installing kubectl on Linux:
...
+To download a specific version, replace the `$(curl -L -s https://dl.k8s.io/release/stable.txt)` portion of the command with the specific version.
 
-   For example, to download version {{< param "fullversion" >}} on Linux, type:
-   -->
-   如需下载某个指定的版本，请用指定版本号替换该命令的这一部分：
-   `$(curl -L -s https://dl.k8s.io/release/stable.txt)`。
+For example, to download version {{< param "fullversion" >}} on Linux, type:
+-->
+如需下载某个指定的版本，请用指定版本号替换该命令的这一部分：
+`$(curl -L -s https://dl.k8s.io/release/stable.txt)`。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/tools/install-kubectl-linux.md:73]:
```suggestion
   For example, to download version {{< param "fullversion" >}} on Linux, type:
   -->
   如需下载某个指定的版本，请用指定版本号替换该命令的这一部分：
   `$(curl -L -s https://dl.k8s.io/release/stable.txt)`。
```

File: content/zh-cn/docs/tasks/tools/install-kubectl-linux.md [hunk 3]
```diff
@@ -64,15 +64,15 @@ The following methods exist for installing kubectl on Linux:
    ```
 
    {{< note >}}
-   <!-- 
-   To download a specific version, replace the `$(curl -L -s https://dl.k8s.io/release/stable.txt)` portion of the command with the specific version.
+<!-- 
+To download a specific version, replace the `$(curl -L -s https://dl.k8s.io/release/stable.txt)` portion of the command with the specific version.
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/tools/install-kubectl-linux.md:68]:
```suggestion
   <!-- 
   To download a specific version, replace the `$(curl -L -s https://dl.k8s.io/release/stable.txt)` portion of the 
   command with the specific version.
```

File: content/zh-cn/docs/tasks/tools/install-kubectl-linux.md [hunk 4]
```diff
@@ -64,11 +64,11 @@ The following methods exist for installing kubectl on Linux:
    ```
 
    {{< note >}}
-   <!-- 
+<!-- 
    To download a specific version, replace the `$(curl -L -s https://dl.k8s.io/release/stable.txt)` portion of the command with the specific version.
 
    For example, to download version {{< param "fullversion" >}} on Linux, type:
-   -->
+-->
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/tools/install-kubectl-linux.md:71]: Don't change this. The current indentation is good.

File: content/zh-cn/docs/tasks/tools/install-kubectl-linux.md [hunk 5]
```diff
@@ -152,14 +152,31 @@ The following methods exist for installing kubectl on Linux:
...
+   -->
+   上面的命令会产生一个警告：
+   ```
+   WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.
+   ```
+   <!--
+   You can ignore this warning. You are only checking the version of `kubectl` that you
+   have installed.
+   -->
+   你可以忽略此警告。你只是通过检查版本证明 `kubectl` 已安装。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/tools/install-kubectl-linux.md:173]:
```suggestion
   你可以忽略此警告。你只是在检查已安装的 `kubectl` 的版本。
```

File: content/zh-cn/docs/tasks/tools/install-kubectl-macos.md [hunk 6]
```diff
@@ -187,6 +187,21 @@ The following methods exist for installing kubectl on macOS:
...
+   -->
+   上面的命令会产生一个警告：
+   ```
+   WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.
+   ```
+   <!--
+   You can ignore this warning. You are only checking the version of `kubectl` that you
+   have installed.
+   -->
+   你可以忽略此警告。你只是通过检查版本证明 `kubectl` 已安装。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/tools/install-kubectl-macos.md:202]:
```suggestion
   你可以忽略此警告。你只是在检查已安装的 `kubectl` 的版本。
```

File: content/zh-cn/docs/tasks/tools/install-kubectl-windows.md [hunk 7]
```diff
@@ -122,7 +122,22 @@ The following methods exist for installing kubectl on Windows:
...
+   -->
+   上面的命令会产生一个警告：
+   ```
+   WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.
+   ```
+   <!--
+   You can ignore this warning. You are only checking the version of `kubectl` that you
+   have installed.
+   -->
+   你可以忽略此警告。你只是通过检查版本证明 `kubectl` 已安装。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/tools/install-kubectl-windows.md:137]:
```suggestion
   你可以忽略此警告。你只是在检查已安装的 `kubectl` 的版本。
```

**Comment by mengjiao-liu** [comment]
Please address these comments and PR needs rebase.

**Pull request closed without merging**